v22.1/cockroach-debug-zip.md

```diff
@@ -146,7 +147,7 @@ Generate a debug zip file containing only log files:
 $ cockroach debug zip ./cockroach-data/logs/debug.zip --include-files=*.log
 ~~~
 
-### Redact sensitive information from the logs
+### Redact sensitive information
 
 Example of a log string without redaction enabled:
```

Comment:
Maybe this is a dumb question, but how do you configure what gets redacted? Does it just automatically redact certain fields? you must be able to configure it somehow, but i'm not seeing this anywhere

Reply:
Redaction is in fact not configurable, only toggle-able. But that's a great call out, I'll include explicit mention when I update for the list of specific redactions, thanks!